### DISCUSSION TITLE
ControlNet best paper at ICCV23 (Marr Prize)

### DISCUSSION THREAD

**Discussion by geroldmeisinger** [lllyasviel/ControlNet#552] [General]
https://openaccess.thecvf.com/content/ICCV2023/html/Zhang_Adding_Conditional_Control_to_Text-to-Image_Diffusion_Models_ICCV_2023_paper.html
https://www.thecvf.com/?page_id=413#ICCVBest

**Congratulations!** :)

**Comment by AugmentedRealityCat** [comment]
Congratulations to you @lllyasviel ! 
You and your team deserve this prize more than anyone else.

ControlNet has transformed Stable Diffusion from the cool toy it used to be into the proper working tool it is today. 

And as if this wasn't enough, you shared that amazingly powerful tool for free with the entire world, and the source code for it as well. This is nothing less than heroic !

**Comment by geroldmeisinger** [comment]
Dall-E 2 showed us what generative AI can do.
Stable Diffusion showed us that it can be done on consumer hardware.
ControlNet showed us that the right extension can give an open-source tool _the canny edge_.
From mere stock image generator to a serious graphics software.
Thank you!
